guides/_posts/2014-01-5-vfl.html

```diff
@@ -88,29 +100,29 @@
 
```

Comment:
I think this is still true, but a default value is no longer defined (I believe it used to be 8).

@d4tocchini and I had discussed throwing an error when using a hyphen without the gap function.

I think we should get finalize our position on this.